client/hotrod-client/src/main/java/org/infinispan/client/hotrod/configuration/AuthenticationConfigurationBuilder.java

```diff
@@ -173,20 +175,31 @@ public AuthenticationConfigurationBuilder password(char[] password) {
 
    /**
     * Specifies the realm to be used for authentication. Username and password also need to be supplied. If none
-    * is specified, this defaults to 'ApplicationRealm'.
+    * is specified, this defaults to 'default'.
```

Comment:
can we call the Constant from this doc ?

Reply:
Done